### PR TITLE
Saleor 5587 create missing test for collections

### PR DESCRIPTION
**Pull request by Kjaroszew**
I want to merge this change because i added missing tests to collection.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

File: cypress/integration/catalog/collections.js [hunk 1]
```diff
@@ -216,7 +219,127 @@ filterTests({ definedTags: ["all"] }, () => {
       });
     });
 
-    it("should update collection", () => {
+    it("As an admin I should be able to delete several collections on collections list page. SALEOR_0309", () => {
```
Comment (eCzerniak) [inline cypress/integration/catalog/collections.js:222]:
```suggestion
    it("to delete several collections on collections list page. SALEOR_0309", () => {
```
or smg different, but without this part "As an admin.."

File: cypress/integration/catalog/collections.js [hunk 2]
```diff
@@ -150,7 +153,7 @@ filterTests({ definedTags: ["all"] }, () => {
         });
     });
 
-    it("should display products hidden in listing", () => {
+    it("Create published collection with products hidden in listings. C:SALEOR_0304", () => {
```
Comment (eCzerniak) [inline cypress/integration/catalog/collections.js:156]:
```suggestion
    it("create published collection with products hidden in listings. C:SALEOR_0304", () => {
```

File: cypress/integration/catalog/collections.js [hunk 3]
```diff
@@ -121,7 +124,7 @@ filterTests({ definedTags: ["all"] }, () => {
         });
     });
 
-    it("should not display collection not set as available in channel", () => {
+    it("Create collection not available for channel. TC:SALEOR_0303", () => {
```
Comment (eCzerniak) [inline cypress/integration/catalog/collections.js:127]:
```suggestion
    it("create collection not available for channel. TC:SALEOR_0303", () => {
```

File: cypress/integration/catalog/collections.js [hunk 4]
```diff
@@ -216,7 +219,127 @@ filterTests({ definedTags: ["all"] }, () => {
...
+        getCollection({ collectionId: firstCollection.id, auth: "auth" })
+          .its("collection")
+          .should("be.null");
+        getCollection({ collectionId: secondColelction.id, auth: "auth" })
+          .its("collection")
+          .should("be.null");
+      });
+    });
+
+    it("As an admin I want to assign product to collection. SALEOR_0307", () => {
```
Comment (eCzerniak) [inline cypress/integration/catalog/collections.js:258]:
```suggestion
    it("assign product to collection. SALEOR_0307", () => {
```

File: cypress/integration/catalog/collections.js [hunk 5]
```diff
@@ -216,7 +219,127 @@ filterTests({ definedTags: ["all"] }, () => {
...
+              .its("collection.products.edges")
+              .should("have.length", 1)
+              .then(producArray => {
+                expect(producArray[0].node.id).to.equal(productToAssign.id);
+              });
+          });
+      });
+    });
+
+    it("As an admin I want to remove product from collection. SALEOR_0308", () => {
```
Comment (eCzerniak) [inline cypress/integration/catalog/collections.js:293]:
```suggestion
    it("remove product from collection. SALEOR_0308", () => {
```

**Review by eCzerniak — APPROVED** [review]
please correct these small things and it is ready to merge.

File: cypress/integration/catalog/collections.js (hunk 1 above)
Comment (Kjaroszew) [reply cypress/integration/catalog/collections.js:222]: corrected

File: cypress/integration/catalog/collections.js (hunk 3 above)
Comment (Kjaroszew) [reply cypress/integration/catalog/collections.js:127]: corrected

File: cypress/integration/catalog/collections.js (hunk 4 above)
Comment (Kjaroszew) [reply cypress/integration/catalog/collections.js]: corrected

File: cypress/integration/catalog/collections.js (hunk 5 above)
Comment (Kjaroszew) [reply cypress/integration/catalog/collections.js]: corrected

File: cypress/integration/catalog/collections.js [hunk 6]
```diff
@@ -216,7 +222,125 @@ filterTests({ definedTags: ["all"] }, () => {
...
+          .get(collectionRow(secondCollection.id))
+          .find(BUTTON_SELECTORS.checkbox)
+          .click()
+          .get(BUTTON_SELECTORS.deleteIcon)
+          .click()
+          .addAliasToGraphRequest("CollectionBulkDelete")
+          .get(BUTTON_SELECTORS.submit)
+          .click()
+          .waitForRequestAndCheckIfNoErrors("@CollectionBulkDelete");
+        // .wait(4000);CollectionBulkDelete
```
Comment (karola312) [inline cypress/integration/catalog/collections.js:252]: Remove comment

**Review by karola312 — APPROVED** [review]

**Pull request closed without merging**